wdl-lint/tests/lints/shellcheck-error/source.errors

```diff
@@ -0,0 +1,48 @@
+note[CommandSectionShellCheck]: `shellcheck` reported the following diagnostic
+   ┌─ tests/lints/shellcheck-error/source.wdl:18:10
+   │
+18 │       if [ -f "$broken"]
+   │          ^ SC1073[error]: Couldn't parse this test expression. Fix to allow more checks.
+   │
+   = fix: address the diagnostics as recommended in the message
```

Comment:
When I use SC on the CL, it usually includes a URL to read more about the rule. The SC docs site is pretty great, and I think we want to include that link if possible (and eventually create _our own_ doc site that's similar).